scripts/bert/data/preprocessing_utils.py

```diff
@@ -0,0 +1,361 @@
+"""Utility classes and functions for data processing"""
+
+__all__ = [
+    'truncate_seqs_equal', 'concat_sequences', 'tokenize_and_align_positions',
+    'get_doc_spans', 'align_position2doc_spans', 'improve_answer_span',
+    'check_is_max_context', 'convert_squad_examples'
+]
+
+import collections
+import itertools
+import numpy.ma as ma
+
+
+def truncate_seqs_equal(seqs, max_len):
```

Comment:
This is specific to BERT? Do we want to include it in the official data APIs?

Reply:
It's not specific to BERT, but currently we only have BERT and XLNet using it.. Not sure if other incoming models need this function.